modules/graphql_views/src/Plugin/Deriver/ViewDeriver.php

```diff
@@ -128,7 +128,7 @@ public function getDerivativeDefinitions($basePluginDefinition) {
         'view' => $viewId,
         'display' => $displayId,
         'paged' => $paged,
-        'cache_tags' => $view->getCacheTags(),
+        'cache_tags' => $view->getExecutable()->getCacheTags(),
```

Comment:
But yeah, that's missing too currently. The schema should not change (should not be invalidated) based on node_list. So this PR actually targets the wrong place. The reason why it works is because the full schema cache tags are also included in the result cache tags. Because obviously, if the schema changes, we also have to invalidate the results generated by that (outdated) vresion of the schema.